rosbag2_py/README.md

```diff
@@ -0,0 +1,20 @@
+# rosbag2_py
+
+## Regenerating Python stub files (.pyi)
+
+Python stub files allow to supply type-hinting information for binary Python modules (e.g. pybind-based).
+
+In rosbag2_py stub files are generated with utility called `stubgen`.
+
+To regenerate stub files
+```
+cd <workspace>
+colcon build --packages-up-to rosbag2_py
+source install/setup.sh
+# Make sure rosbag2_py can be imported
+python3 -c 'import rosbag2_py'
+
+sudo apt update && sudo apt install mypy
```

Comment:
There is no real reason to pin it. The version we get from `apt` should not change for Ubuntu 22.04. And when ROS 2 switches to Ubuntu 24.04 (in the next few months), we'll want to use that version: https://packages.ubuntu.com/noble/mypy.

Reply:
Yes. And I want it to be pinned explicitly in both places. Otherwise transition period will be a nightmare.
As soon we will need to switch to the new version on the mypy we will update github script and readme file simultaneously to be in synch.

Reply:
@MichaelOrlov fixed

Reply:
I don't think pinning will change anything, because usually there's only one version available from the Ubuntu system packages and it doesn't change for a given Ubuntu version.

Reply:
I also don't see a lot of benefits of pinning the version. Instead, we can add to readme that we use version from apt repositories for the supported Ubuntu version